### ISSUE TITLE
Hulk performance in Figure 3.

### ISSUE THREAD

**Issue by diamond0910** [threedle/text2mesh#10]
I train the code and use the default script 'python main.py --run branch --obj_path data/source_meshes/person.obj --output_dir results/demo/people/hulk --prompt "a 3D rendering of the Hulk in unreal engine" --sigma 12.0  --clamp tanh --n_normaugs 4 --n_augs 1 --normmincrop 0.1 --normmaxcrop 0.4 --geoloss --colordepth 2 --normdepth 2 --frontview --frontview_std 4 --clipavg view --lr_decay 0.9 --clamp tanh --normclamp tanh  --maxcrop 1.0 --save_render --seed 23 --n_iter 1500  --learning_rate 0.0005 --normal_learning_rate 0.0005 --standardize --no_pe --symmetry --background 1 1 1'

Finally, I got the following result which is different from the paper. The face is very unclear, the color of the pants is green, and the overall clarity is not as good as in the paper. Where did I go wrong and what should I do? 

my result:
![image](https://user-images.githubusercontent.com/43036573/149161911-d0529310-5c14-412c-ac26-3778ee3b35af.png)

paper:
![image](https://user-images.githubusercontent.com/43036573/149160386-acbbef45-5ef8-4566-9e04-cfd5ef107628.png)

Best

**Comment by diamond0910** [comment]
Is there any reply?

Thank you
